rtloader/three/three.h

```diff
@@ -67,6 +67,7 @@ class Three : public RtLoader
                   RtLoaderPyObject *&check);
 
     char *runCheck(RtLoaderPyObject *check);
+    void cancelCheck(RtLoaderPyObject *check);
```

Comment:
Maybe out of the scope of the PR, but using the `override` keyword would be nice: https://en.cppreference.com/w/cpp/language/override.